### PR TITLE
ch(mock file):add mock data and remove dummy data [start #168895124]

### PR DESCRIPTION
**Pull request by mwafrika**
#### What does this PR do?
Add mock data for the test, remove data in the data structure and add the status code
#### Description of Task to be completed?
- Add mock folder then add files containing dummy data inside the folder.
- Add appropriate status code for each status
- import the folder in the testing file
- detect the position of  the array to be used 

#### How should this be manually tested?
The testing files should be cleaned.
status code should be specific to the message

#### What are the relevant pivotal tracker stories?
#168895124
#### Questions:
How can I exclude the testing files for the coverage so that they cannot affect the coverage status ??

### CODE REVIEW

File: Server/services/userService.js [hunk 1]
```diff
@@ -43,13 +40,14 @@ class userHelper {
 
   static signin(userData) {
     const allData = db.users;
+    // eslint-disable-next-line max-len
```
Comment (Kasulejoseph) [inline Server/services/userService.js]: Configure all the eslint rules from the eslint file

File: Server/controllers/articleController.js [hunk 2]
```diff
@@ -37,27 +36,28 @@ const artController = {
...
+    const comment = db.comments.filter((com) => com.artID === artid);
+    const noComment = [];
+    const commToSend = comment.length > 0 ? comment : noComment;
+    const findArticle = await artService.getSpecific(id, commToSend);
     if (!findArticle) return res.status(404).send({ status: 'error', error: 'Article not found, please try another' });
 
     return res.status(200).send({
-      status: 'success',
+      status: 200,
       data: findArticle,
```
Comment (Kasulejoseph) [inline Server/controllers/articleController.js:48]: This should have `message` as a key as well

File: Server/controllers/userController.js [hunk 3]
```diff
@@ -16,8 +16,12 @@ class userController {
       });
     }
     return jwt.sign(signup, process.env.JWT_SECRET, (err, token) => {
+      // console.log(signup);
```
Comment (Kasulejoseph) [inline Server/controllers/userController.js]: Remove all the debugger statements

File: Server/controllers/userController.js [hunk 4]
```diff
@@ -26,14 +30,17 @@ class userController {
...
-        error: 'invalid credentials!!! no user exist',
+      res.status(401).send({
+        status: '401',
+        error: 'invalid credentials!!',
       });
     }
     return jwt.sign(myUsers, process.env.JWT_SECRET, (err, token) => {
       res.status(200).send({
+        status: 200,
+        myUsers: 'successfully logged in',
```
Comment (Kasulejoseph) [inline Server/controllers/userController.js]: This should be message not `myUsers`

File: Server/controllers/articleController.js (hunk 2 above)
Comment (mwafrika) [reply Server/controllers/articleController.js:48]: fixed alredy

File: Server/controllers/userController.js (hunk 3 above)
Comment (mwafrika) [reply Server/controllers/userController.js]: already fixed.

File: Server/controllers/userController.js (hunk 4 above)
Comment (mwafrika) [reply Server/controllers/userController.js]: fixed already

**Review by Kasulejoseph — APPROVED** [review]
Looks good

**Pull request merged**